### ISSUE TITLE
Long press on menu button makes "click click click"

### ISSUE THREAD

**Issue by schmurtzm** [OnionUI/Onion#952]
### Checklist

- [X] I am reporting a bug in the Onion OS (Note that Onion devs can’t fix inherited issues with RA Cores)
- [X] I have reviewed the [Onion Wiki](https://github.com/OnionUI/Onion/wiki) and did not find the answer
- [X] I have searched the existing (open and closed) issues to ensure I do not log a duplicate

### Onions OS Version

4.2.0

### Miyoo Firmware Version

0609

### Provide a clear and concise description of the issue

Long press on menu button makes "click click click" , it was disabled before in 4.1.0

it comes from the keymon which runs sendkeys commands. I've got the culprit :
https://github.com/OnionUI/Onion/blob/79c284084ade242a5217d97d6f861f50bd7108af/src/keymon/menuButtonAction.h#L110

It's frustrating because replacing L2 by L1 here doesn't help to suppress the "clic clic clic"  : https://github.com/OnionUI/Onion/blob/79c284084ade242a5217d97d6f861f50bd7108af/src/keymon/menuButtonAction.h#L110
But making this on UART during pressing menu button stops the "clic clic clic":
/mnt/SDCARD # ./.tmp_update/bin/sendkeys 18 1
/mnt/SDCARD # ./.tmp_update/bin/sendkeys 18 0

I see that quietMainUI() is repeating but it seems that I'm not able to find a way to execute it once at the right time.

To explain the change of behavior
On old MainUI :  L1 = page up / click sound   & L2 = nothing / no sound
On new MainUI :  L1 = nothing / click sound   & L2 = page up / no sound

In other words Miyoo has remaped the key without remaping the sound. It explains why triggering L1 a lot of times in keymon triggers a "clic clic clic" sound too.

So the idea is to triggers only one time an artificial L1 once we are in long menu press (it should trigger only one second clic which is not bad to indicate a long press)

### Steps to Reproduce

Long press on menu button makes "click click click" , during this time press one shoulder button to stop the click, that's what we try to reproduce.

### Provide any additional context or information that may help us investigate your issue

https://github.com/OnionUI/Onion/blob/79c284084ade242a5217d97d6f861f50bd7108af/src/keymon/menuButtonAction.h#L110
